src/dotnet/commands/dotnet-add-p2p/Program.cs

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+
+namespace Microsoft.DotNet.Tools.Add.ProjectToProjectReference
+{
+    public class AddProjectToProjectReferenceCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "dotnet add p2p",
```

Comment:
@blackdwarf Have you reviewed the strings here?
@piotrpMSFT We need to have these localized. Do we still have time for that?